go/test/endtoend/vreplication/multi_tenant_test.go

```diff
@@ -190,6 +190,7 @@ func TestMultiTenantSimple(t *testing.T) {
 	for _, ks := range []string{sourceKeyspace, sourceAliasKeyspace} {
 		lastIndex = insertRows(lastIndex, ks)
 	}
+	waitForWorkflowState(t, vc, fmt.Sprintf("%s.%s", targetKeyspace, mt.workflowName), binlogdatapb.VReplicationWorkflowState_Running.String())
```

Comment:
This is unrelated, but addresses a test flake that was observed on the PR CI runs.